wgengine/router/router_linux.go

```diff
@@ -1400,6 +1400,15 @@ func normalizeCIDR(cidr netip.Prefix) string {
 // The function calls cleanUp for both iptables and nftables since which ever
 // netfilter runner is used, the cleanUp function for the other one doesn't do anything.
 func cleanUp(logf logger.Logf, interfaceName string) {
+	if distro.Get() == distro.Synology {
```

Comment:
nit: create a helper like `isNetfilterSupported` and reuse here and in `setNetfilterMode`?

Reply:
done